Meadowlark-js/backends/meadowlark-elasticsearch-backend/src/repository/ElasticSearchException.ts

```diff
@@ -21,7 +21,21 @@ export async function handleElasticSearchError(
     if (elasticSearchClientError?.name !== undefined) {
       switch (elasticSearchClientError.name) {
         case 'ConfigurationError':
-        case 'ConnectionError':
+        case 'ConnectionError': {
+          if (elasticSearchClientError?.message !== undefined) {
+            Logger.error(
+              `${moduleName} ${documentProcessError}`,
+              traceId,
+              `(${elasticSearchClientError.name}) - ${elasticSearchClientError.message}`,
+            );
+            return {
+              response: 'QUERY_FAILURE_CONNECTION_ERROR',
+              documents: [],
+              failureMessage: elasticSearchClientError.message,
+            };
+          }
+          break;
+        }
         case 'DeserializationError':
         case 'NoLivingConnectionsError':
         case 'NotCompatibleError':
```

Comment:
It looks like a bunch of these are also connection issues. In general, `QUERY_FAILURE_INVALID_QUERY` means our Meadowlark code did something wrong and a 500 should be returned. That's taken care of in the ResponseError handling.

The rest of these are either Elasticsearch, network or Elasticsearch client library issues. Elasticsearch and network issues should return `QUERY_FAILURE_CONNECTION_ERROR` and the rest `UNKNOWN_ERROR`.

`QUERY_FAILURE_CONNECTION_ERROR`: ConfigurationError, ConnectionError, RequestAbortedError, TimeoutError
`UNKNOWN_ERROR`: DeserializationError, NoLivingConnectionsError, NotCompatibleError, ElasticsearchClientError, SerializationError